src/analysis/inner_product_space/basic.lean

```diff
@@ -1066,6 +1066,43 @@ begin
   simp only [sq, ← mul_div_right_comm, ← add_div]
 end
 
+section complex
+
+variables (V : Type*)
```

Comment:
```suggestion
variables {V : Type*}
```
There is no reason to make `V` an explicit parameter here, it can be inferred from the other parameters.

Reply:
I see, will use implicit when it makes sense.